src/main/java/io/antmedia/AntMediaApplicationAdapter.java

```diff
@@ -1377,6 +1377,9 @@ public static boolean updateAppSettingsFile(String appName, AppSettings newAppse
 		
 		store.put(AppSettings.SETTINGS_VOD_UPLOAD_FINISH_SCRIPT, newAppsettings.getVodFinishScript() != null ? String.valueOf(newAppsettings.getVodFinishScript()) : "");
 		
+		store.put(AppSettings.SETTINGS_DASH_FRAGMENT_DURATION, String.valueOf(newAppsettings.getDashFragmentDuration()));
+		store.put(AppSettings.SETTINGS_DASH_SEG_DURATION, String.valueOf(newAppsettings.getDashSegDuration()));
```

Comment:
getDashSegDuration and getDashFramentDuration are already String. Why are we using `String.valueOf`